test/module/irohad/ametsuchi/flat_file_test.cpp

```diff
@@ -122,3 +126,71 @@ TEST_F(BlStore_Test, BlockStoreInitializationFromNonemptyFolder){
   // check that last ids of both block storages are the same
   ASSERT_EQ(bl_store1->last_id(), bl_store2->last_id());
 }
+
+TEST_F(BlStore_Test, EmptyDumpDir) {
+  auto res = check_consistency("");
+  ASSERT_EQ(res, nonstd::nullopt);
+}
+
+TEST_F(BlStore_Test, GetNonExistingFile) {
+  auto bl_store = FlatFile::create(block_store_path);
+  Identifier id = 98759385;  // random number that will not exist
+  auto res = bl_store->get(id);
+  ASSERT_EQ(res, nonstd::nullopt);
+}
+
+TEST_F(BlStore_Test, GetDirectory) {
+  auto bl_store = FlatFile::create(block_store_path);
+  ASSERT_EQ(bl_store->directory(), block_store_path);
+}
+
+TEST_F(BlStore_Test, GetDeniedBlock) {
+  std::vector<uint8_t> block(100000, 5);
```

Comment:
Used pretty often, so you may extract it to the fixture (and rename to blob or something like that)

Reply:
Fixed